spring-cloud-generator/generate-library-list.sh

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
+mkdir -p run-sanity-check
+touch run-sanity-check/generate-library-list-started
```

Comment:
I think the library list being created is a sufficient side effect that we can check.

Reply:
Added library list checks in `sanity-checks.sh`